lib/ssh/src/ssh_server_channel.erl

```diff
@@ -51,8 +51,7 @@ tree.
 > When implementing a client subsystem handler, use
 > [\-behaviour(ssh_client_channel)](`m:ssh_client_channel`) instead.
 """.
--moduledoc(#{since => "OTP 21.0",
```

Comment:
Callbacks should have title callbacks, same comment as previously.

Reply:
see reply above.